Add individual Markdown file generation for each page (#28) (#32)
delucis/starlight-llms-txt: docs/src/content/docs/getting-started.mdx

description: How to install and set-up starlight-llms-txt in your Starlight docs site
---

> `starlight-llms-txt` is a plugin for the [Starlight](https://starlight.astro.build/) documentation website framework that auto-generates `llms.txt`, `llms-full.txt`, and `llms-small.txt` context files for large language models based on your documentation site's content. It also generates individual Markdown files for each page, fully implementing the [llmstxt.org standard](https://llmstxt.org/#proposal).

Reviewer: Nit, but “can” here probably makes sense given it’s opt in:

```suggestion
`starlight-llms-txt` is a plugin for the [Starlight](https://starlight.astro.build/) documentation website framework that auto-generates `llms.txt`, `llms-full.txt`, and `llms-small.txt` context files for large language models based on your documentation site's content. It can also generate individual Markdown files for every docs page.
```

Author: Addressed in c1e140d.

   <PackageManagers type="run" args="dev" />

> 4. Visit [`localhost:4321/llms.txt`](http://localhost:4321/llms.txt) to preview the plugin in action. Individual page Markdown files are also available at URLs like [`localhost:4321/getting-started.md`](http://localhost:4321/getting-started.md).

Reviewer: And probably remove this as if someone followed just the introductory set-up steps here, this link would 404 because individual files aren’t set up yet:

```suggestion
4. Visit [`localhost:4321/llms.txt`](http://localhost:4321/llms.txt) to preview the plugin in action.
```
